src/pages/project/project_wizard/ProjectForm.tsx

```diff
@@ -38,10 +38,10 @@ export const ProjectForm = ({ project }: ProjectFormProps) => {
               <RequiredDescription />
               <Box sx={{ bgcolor: 'secondary.dark', padding: '0' }}>
                 <Box id="form" sx={{ bgcolor: 'secondary.main', mb: '2rem', padding: '1.5rem 1.25rem' }}>
-                  {tabNumber === ProjectTab.Description && <ProjectDescriptionForm project={project} />}
-                  {tabNumber === ProjectTab.Details && <ProjectDetailsForm />}
-                  {tabNumber === ProjectTab.Contributors && <ProjectContributorsForm />}
-                  {tabNumber === ProjectTab.Connections && <ProjectConnectionsForm />}
+                  {tabNumber === ProjectTabs.Description && <ProjectDescriptionForm project={project} />}
+                  {tabNumber === ProjectTabs.Details && <ProjectDetailsForm project={project} />}
```

Comment:
Her sendes `project`, som er responsen fra API-kallet som gjøres først. Den vil dermed ikke ha oppdaterte verdier som brukeren har lagt inn etterpå. Lurer derfor på om vi heller skulle tatt vekk det, og bruke `const { values } = useFormikContext<CristinProject>()` i de komponenten som trenger data fra prosjektet?

```suggestion
                  {tabNumber === ProjectTabs.Description && <ProjectDescriptionForm />}
                  {tabNumber === ProjectTabs.Details && <ProjectDetailsForm />}
```